musicname/ze_shroomforest_v4_5.cfg

```
@@ -1,21 +1,21 @@
     "jazz jackrabbit - title theme.mp3"                                         "Jazz Jackrabbit - Title Theme"
     "jazz jackrabbit 2 - dark groove.mp3"                                       "Jazz Jackrabbit 2 - Dark Groove"
     "john 5 - black widow of la porte.mp3"                                      "John 5 - Black Widow Of La Porte"
-    "mapclear.mp3"                                                              "Super Mario 64 - Ultimate Bowser Clear"
-    "mario kart wii music - trophy presentations.mp3"                           "Mario Kart Wii Music - Trophy Presentations"
+    "mapclear.mp3"                                                              "Super Mario 64 OST - Ultimate Bowser Clear"
+    "mario kart wii music - trophy presentations.mp3"                           "Mario Kart Wii OST - Trophy Presentations"
     "megadeth - duke nukem theme.mp3"                                           "Megadeth - Duke Nukem Theme"
-    "opening - donkey kong.mp3"                                                 "Opening - Donkey Kong"
+    "opening - donkey kong.mp3"                                                 "Super Smash Bros. Wii U - Opening (Donkey Kong)"
     "pokefloats.mp3"                                                            "Super Smash Bros. Melee - Poké Floats"
-    "pokemon - reshiram & zekrom.mp3"                                           "Pokemon - Reshiram & Zekrom"
-    "pokemon gameboy music.mp3"                                                 "Pokemon - Gym Leader Battle Theme"
-    "stickerbrush symphony.mp3"                                                 "Donkey Kong Country 2 - Stickerbush Symphony"
-    "super paper mario music_ mimi battle.mp3"                                  "Super Paper Mario Music - Mimi Battle"
-    "super paper mario soundtrack - evil king bowser_ here!.mp3"                "Super Paper Mario Music - Evil King Bowser Here"
-    "the fellowship of the ring soundtrack-02-concerning hobbits.mp3"           "The Fellowship of the Ring- Concerning Hobbits"
-    "tough guy alert!.mp3"                                                      "Mario & Luigi Bowser's Inside Story - Though Guy Alert!"
+    "pokemon - reshiram & zekrom.mp3"                                           "Pokemon Black & White OST - Reshiram & Zekrom"
+    "pokemon gameboy music.mp3"                                                 "Pokemon Gameboy OST - Gym Leader Battle Theme"
+    "stickerbrush symphony.mp3"                                                 "Donkey Kong Country 2 OST - Stickerbush Symphony"
+    "super paper mario music_ mimi battle.mp3"                                  "Super Paper Mario OST - Mimi Battle"
+    "super paper mario soundtrack - evil king bowser_ here!.mp3"                "Super Paper Mario OST - Evil King Bowser Here"
+    "the fellowship of the ring soundtrack-02-concerning hobbits.mp3"           "The Fellowship of the Ring - Concerning Hobbits"
```

Add LOTR to the name?
```suggestion
    "the fellowship of the ring soundtrack-02-concerning hobbits.mp3"           "LOTR: The Fellowship of the Ring OST - Concerning Hobbits"
```